docs/production-deployment/cloud/high-availability/enable.mdx

```diff
@@ -121,49 +121,63 @@ Temporal Cloud can't change replica locations directly.
 To update the location, you need to remove the current replica and add a new one.
 Follow these steps to change the replica location:
 
-1. [Remove your replica](#discontinuing).
+1. [Remove your replica](#disable).
    This disables High Availability for your Namespace.
 2. Wait through the required 7-day waiting period.
 3. [Add a new replica](#upgrade) to your Namespace.
 
 You will receive an email alert once your Namespace is ready for use.
 
-## Discontinue High Availability features {#discontinuing}
+## Disable High Availability (remove a replica) {#disable}
 
-Removing a Namespace replica disables High Availability and automatic failover features.
-Follow these steps to disable these features and end High Availability charges:
+To disable High Availability features on a Namespace, remove the replica from that Namespace. Removing a replica disables all High Availability features: 
+
+- Discontinues replication of the Workflows in the Namespace.
+- Disables the Namespace's ability to trigger a failover to a different region or cloud.
+- For Workers and Clients that use API keys, requires connecting to the Namespace using the published [regional endpoint](/cloud/regions) for the Namespace's region.
+    - Disables connecting to the Namespace with API keys and the Namespace's endpoint or the replica region's regional endpoint.
+- Ends High Availability charges.
+
+Follow these steps to remove a replica from a Namespace:
 
 <Tabs>
 
 <TabItem value="webui" label="Web UI">
 
+1. If using API keys on this Namespace, configure Workers and Clients that use API keys to [connect with the regional Temporal Cloud endpoint](/cloud/api-keys#namespace-authentication) for the Namespace's primary region.
 1. Navigate to the Namespace details page in Temporal Cloud
 1. Select the option to "Remove Replica" on the "Region" card.
 
 </TabItem>
 
 <TabItem value="tcldcli" label="tcld">
 
-At the command line, enter:
+First, if using API keys on this Namespace, configure Workers and Clients that use API keys to [connect with the regional Temporal Cloud endpoint](/cloud/api-keys#namespace-authentication) for the Namespace's primary region.
+
+Then, at the command line, enter:
 
 ```
 tcld namespace delete-region \
     --namespace <namespace_id>.<account_id> \
     --region <replica_region>
 ```
 
-If using API key authentication with the `--api-key` flag, you must add it directly after the tcld command and before `namespace delete-region`
+If using the `--api-key` flag to issue this CLI command, you must add it directly after the tcld command and before `namespace delete-region`
```

Comment:
What does this mean to "Add it directly after the `tcld` command?" This seems to suggest there is more than 1 command, but we are only showing one command. Do we mean that they must put the `--api-key` flag in the command?